features/dryrun_mode.feature

```diff
@@ -1,3 +1,4 @@
+@dryrun
```

Comment:
Do we need this? You should be able to run this feature only by targeting the file

Reply:
Yeah, true, removed